admin_manual/configuration_files/external_storage/amazons3.rst

```diff
@@ -20,9 +21,12 @@ highly-recommended.
 .. figure:: images/amazons3.png
    :alt:
 
-Optionally, you can override the hostname, port and region of your S3 server,
+Optionally, you can override the hostname, and port of your S3 server,
 which is required for non-Amazon servers such as Ceph Object Gateway.
 
+**Region** should be an [AWS Region Code](https://docs.aws.amazon.com/general/latest/gr/rande.html#region-names-codes).
+If no region is supplied, a new bucket will be created in the `eu-west-1` region.
```

Comment:
@juliushaertl this is an ongoing issue that, should probably get added. With how it's coded, `eu-west-1` is **always used and for all s3-providers**, unless the user specifies the region.

Ideally, "auto" is used and not `eu-west-1`. But in the meantime, the docs need to make it clear that "region" is a required field. Even if you use another S3-compatible provider (like I'm doing with Cloudflare R2"), Nextcloud will still try to use `eu-west-1`.

Reply:
Alright, I created #12098 to help clarify this.